flames/selenium/docker-compose.override.yml

```diff
@@ -0,0 +1,52 @@
+version: '2.1'
+
+services:
+
+    selenium-hub:
+        image: elgalu/selenium:${SELENIUM_SERVER:-latest}
+        hostname: ${HOST_NAME_SELENIUM:-selenium-hub}
```

Comment:
I would not variablize it, as none of the other provided services are variablized. Just to adhere to the current standard. Simply use `selenium` (Let's discuss further if there are [naming collisions](https://github.com/devilbox/flames/pull/1/files#r280974813))

Reply:
Main reason why I've variablized `HOST_NAME_SELENIUM` is that we should change on each environment call to `SELENIUM_HUB_HOST` entry (for hub, chrome and/or firefox, if we used it's client too). That's means two or three times !